src/vs/editor/contrib/parameterHints/parameterHints.css

```diff
@@ -96,7 +96,7 @@
 	text-align: center;
 	height: 12px;
 	line-height: 12px;
-	opacity: 0.5;
+	opacity: 0.1;
```

Comment:
My understanding of the issue is that the opacity needs to be increased, not decreased.

Reply:
Thank you for the feedback.
Yes, I changed that.